reflectometry/amor_reduction.ipynb

```diff
@@ -225,14 +227,47 @@
     "sc.plot(lambda_theta)"
    ]
   },
+  {
+   "cell_type": "code",
```

Comment:
If this is not really urgent, can we wait until this is merged in `ess`? https://github.com/scipp/ess/pull/23

Reply:
It 100 % can wait. Just wanted to put it in when it was on the top of my head.